### ISSUE TITLE
Uninitialized constant Contentful::Importer::Command::VERSION

### ISSUE THREAD

**Issue by pdrosos** [contentful/contentful-importer.rb#20]
Hello,

I see this error when trying to execute 

```
contentful-importer --configuration=settings.yml import
```
```
/usr/local/rvm/gems/ruby-1.9.3-p551/gems/contentful-importer-0.2.0/lib/contentful/importer/command.rb:35:in `<class:Command>': uninitialized constant Contentful::Importer::Command::VERSION (NameError)
        from /usr/local/rvm/gems/ruby-1.9.3-p551/gems/contentful-importer-0.2.0/lib/contentful/importer/command.rb:21:in `<module:Importer>'
        from /usr/local/rvm/gems/ruby-1.9.3-p551/gems/contentful-importer-0.2.0/lib/contentful/importer/command.rb:10:in `<module:Contentful>'
        from /usr/local/rvm/gems/ruby-1.9.3-p551/gems/contentful-importer-0.2.0/lib/contentful/importer/command.rb:9:in `<top (required)>'
        from /usr/local/rvm/rubies/ruby-1.9.3-p551/lib/ruby/site_ruby/1.9.1/rubygems/core_ext/kernel_require.rb:54:in `require'
        from /usr/local/rvm/rubies/ruby-1.9.3-p551/lib/ruby/site_ruby/1.9.1/rubygems/core_ext/kernel_require.rb:54:in `require'
        from /usr/local/rvm/gems/ruby-1.9.3-p551/gems/contentful-importer-0.2.0/bin/contentful-importer:10:in `<top (required)>'
        from /usr/local/rvm/gems/ruby-1.9.3-p551/bin/contentful-importer:23:in `load'
        from /usr/local/rvm/gems/ruby-1.9.3-p551/bin/contentful-importer:23:in `<main>'
        from /usr/local/rvm/gems/ruby-1.9.3-p551/bin/ruby_executable_hooks:15:in `eval'
        from /usr/local/rvm/gems/ruby-1.9.3-p551/bin/ruby_executable_hooks:15:in `<main>'
```

Could you please help me to resolve this and run the importer?
I am on Debian Wheezy with ruby 1.9.3p551 installed. Each command I try gives me the same error.

Thanks in advance

**Comment by neonichu** [comment]
This is actually a bug in version 0.2.0 — we'll fix it and release a new version. Thanks for reporting!

**Comment by pdrosos** [comment]
Thanks for the information!

Will it be released soon, or it's better to use the previous version? I just installed it and it works. I need to make an import these days.

And a bit off-topic question, instead of opening a new issue - is there a way to update already imported entries? I have tried to import some entries and then I added a new entry, changed one of the already imported and have run the import again. The newly added entry was imported, but the changed one was not updated. What would be the best way to handle such situation?

Thanks in advance!

**Comment by neonichu** [comment]
It should be a quick fix, but unless you need any of the things added in 0.2.0 (see <https://github.com/contentful/contentful-importer.rb/blob/master/CHANGELOG.md#020>), you will also be fine using the previous version.

To your other question: this is meant as a one time import, so when you run a second import, any resources which have a counterpart with the same UID already in the system will be skipped. The easiest way to handle this would probably be to both delete the various log files (because they're used to keep track of already imported data) as well as the space you're importing to.

**Comment by pxlpnk** [comment]
@pdrosos We just released a new version (0.2.1) that fixes your issue.
